src/TemplateRenderer.php

```diff
@@ -147,34 +152,59 @@ public function renderSimpleDatasetSiteFromDataDownloads(
             "datePublished" => $settings["dateFirstPublished"],
         ]);
 
+        if (isset($settings["openBookingAPIBaseUrl"])) {
+            $dataset->setAccessService(new WebAPI([
+                "name" => "Open Booking API",
+                "description" => "An API that allows for seamless booking experiences to be created for ".
+                    strtolower($dataFeedHumanisedList)." available from ".
```

Comment:
If this is a list, running `strtolower` will print out `array`. Use `array_map('strtolower', $dataFeedHumanisedList)` instead.

Reply:
Confusingly `$dataFeedHumanisedList` appears to be a string. Have renamed to `$dataFeedListHumanisedString` to clarify.